lib/source-map-generator.js

```diff
@@ -72,6 +72,17 @@ SourceMapGenerator.fromSourceMap =
       generator.addMapping(newMapping);
     });
     aSourceMapConsumer.sources.forEach(function (sourceFile) {
+      if (aSourceMapConsumer._mappings === '') {
```

Comment:
Please remove this if-check. We should always execute the code inside this if-block, even if the mappings are not empty.